packages/core/admin/admin/src/components/MainNav/NavLink.tsx

```diff
@@ -39,8 +41,25 @@ const MainNavLinkWrapper = styled(RouterLink)`
   }
 `;
 
+const getGuidedTourTooltip = (to: To) => {
+  const normalizedTo = to.toString().replace(/\//g, '');
+
+  switch (normalizedTo) {
+    case 'content-manager':
```

Comment:
I suppose you made a typo, is the content type builder case

Reply:
I don't think so because we want to point to the content-manager at the final step of the content-type builder tour